src/views/Home.jsx

```diff
@@ -1,7 +1,41 @@
 import './Home.css';
 import { SingleList } from '../components/SingleList';
+import { useEffect, useState } from 'react';
+import { createList } from '../api/firebase';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, setListPath, user }) {
+	const [shoppingListName, setShoppingListName] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const { uid, email } = user;
+
+		const names = data.map((list) => {
+			return list.name;
+		});
+
+		if (names.includes(shoppingListName)) {
+			alert('name should be unique');
+
+			return;
+		}
+
+		createList(uid, email, shoppingListName);
+		alert('created');
```

Comment:
Not a big deal now, but maybe we add a todo here to revisit notification messages? Might be a nice user experience to have something a bit more descriptive. Same applies to the alert above when checking if a list name exists.